### ISSUE TITLE
adding socioeconomic/demographic measures to western washington prospect list data

### ISSUE THREAD

**Issue by ozanj** [ozanj/rclass#9]
@cyouh95 

for the prospect list from western washington university, the measures of median income and race are at the zip-code level, correct?

how hard would it be to add city-level and state-level measures of?:
- median income [working age population]
- race

this would be used for a problem set I have to create by Friday

**Comment by cyouh95** [comment]
@ozanj yup, they are zip-code data! State-level shouldn't be hard to get. And it looks like city-level should be possible as well, if **312** [here](https://api.census.gov/data/2016/acs/acs5/geography.html) is the correct geographic unit. Just hope merging in the city data won't be too big an issue, if that needs to be done by city name/state?

**Comment by ozanj** [comment]
@cyouh95 @mpatricia01 

Thanks Crystal, can you add the state-level variables. let me know when that is done and we'll start working those vars into the problem set.

after that is done, try adding city vars but if it looks like it will take a long time or if it looks like quality of merge will be bad then don't add them.

thank you!

**Comment by cyouh95** [comment]
@ozanj @mpatricia01 Here is the CSV w/ the state-level variables: 45f9ebf

- `state_name`
- `state_fips_code` (_2-digit code_)
- `pop_total_state`, `pop_white_state`, etc. 
- `median_inc_2544_state`, `median_inc_4564_state` (_might not be needed in final data, but was used to calculated below_)
- `avgmedian_inc_2564_state`

**Comment by cyouh95** [comment]
@ozanj @mpatricia01 Ah, just realized there is a `STAT_CODE`, `ZIP`, `HS_STATE`, and `HS_CITY` in the original data. The home state (`STAT_CODE`) and `HS_STATE` might be different (But I think mostly because many rows are missing the `HS_STATE`/`HS_CITY` fields).

Currently, state-level data is merged to `STAT_CODE`. Zip-code level was to (home) `ZIP` as well, but I guess city-level data would have to be the HS's city?

**Comment by ozanj** [comment]
@cyouh95 @mpatricia01 

Prior to seeing note you just sent, I just pulled changes to wwlist_merged_state.csv 

then I modified create_prospect_list.R and modified wwlist_merged.Rdata and pushed those changes to github.

why don't you check out whether stuff I did still works in light of note you just sent.

**Comment by ozanj** [comment]
@cyouh95 @mpatricia01 

Separate request: can you provide Patricia and I with information about how the prospect list defines race/ethnicity [var=ethn_code] and how ACS data defines race/ethnicity?

Patricia will create lead in developing draft of problem set. and one set of questions will compare the race/ethnicity of prospects purchased [at zip-code level or state level] to overall race/ethnicity composition at the zip-code level or state-level.

a potential concern is that prospect list definition of race/ethnicity may differ from ACS definition, so would be helpful for Patricia to see the definitions so that she can make decisions about what is possible to ask students to do.

**Comment by cyouh95** [comment]
@ozanj @mpatricia01 Everything should still work fine - I think we do want the state-level census data merged to `STAT_CODE` instead of `HS_STATE` right? (since the latter field is missing for more obs) But don't think there's many cases where the states would differ!

But if we can get city-level data, then it'd have to be merged to `HS_CITY`, whereas zip-code (and state-level) are to home location, if that's okay?
